corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/api.js

```diff
@@ -59,7 +59,17 @@ hqDefine("cloudcare/js/formplayer/apps/api", [
                 return null;
             }
             var appCollection = Collections(apps);
-            return appCollection.get(id);
+            let fetchedApp = appCollection.get(id);
```

Comment:
nice catch, nit: I think this would be equivalent?

```
var appCollection = Collections(apps);
let fetchedApp = appCollection.get(id);

if (!fetchedApp) {
    fetchedApp = appCollection.models.find(currApp => currApp.get("copy_of") === id);
}

```

Reply:
yeah, I like that better.